### PR TITLE
Always show full url in table, ref. DCOS-6085

### PR DESCRIPTION
**Pull request by mlunoe**
In collaboration with the designers we came to the conclusion to always show the full URL and break it into multiple lines in the table, if there is not enough space. An added benefit with this is that people now can copy the whole URL if needed.

Also, the URL field will now take up 'the rest of the space' in the table, if there is any left.

Small screen:

Bigger screen:

### CODE REVIEW

**Comment by kennyt** [comment]
if you wanted to save even more space, I don't see why we couldn't cut down the width of the column for the remove button

 px

**Comment by mlunoe** [comment]
I set the 'Remove' column width to be the size of the largest width it will take up at any point in time

File: src/styles/components/typography.less [hunk 1]
```diff
@@ -196,6 +196,14 @@ components/typography.less
 
   }
 
+  .text-overflow-break-word {
+
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    word-break: break-word;
```
Comment (jfurrow) [inline src/styles/components/typography.less]: Do we need all of these properties?

Comment (mlunoe) [inline src/styles/components/typography.less]: This is my source of reference: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/

From their explanation, I understand that all those are necessary.

Comment (daiyiwhale) [inline src/styles/components/typography.less]: Oh wow, all bases covered with these properties ahaha

**Comment by daiyiwhale** [comment]
looks good 👍

**Pull request merged**